src/main/java/com/smartling/connector/eloqua/sdk/rest/api/EmailApi.java

```diff
@@ -13,7 +13,7 @@
     Elements<Email> listEmails(@Param("depth") Depth depth, @Param("page") int page, @Param("count")int count, @Param("orderBy")String orderBy, @Param("search") String search);
 
     @RequestLine("GET /API/REST/2.0/assets/email/folder/{folderId}/contents?depth={depth}&page={page}&count={count}&orderBy={orderBy}&search={search}")
```

Comment:
When i tested by postmat the `search` required value in single quotes. Let's test behavior manually after release.

Reply:
We build search param on next formula `[name =] "*<searchTerm>*"`. So we can pass string with whitespaces as search param and it will work correct